### ISSUE TITLE
NicoVideo embed unable to display

### ISSUE THREAD

**Issue by KyoriAsh** [HydraWiki/mediawiki-embedvideo#64]
After the [https://github.com/HydraWiki/mediawiki-embedvideo/issues/54] issue, it seems all iframe are displayed in blank white instead of contents. Is there any fixed for this issue as IE/Edge seems doesn't accept srcdoc parameter and even blank white for Chrome and Firefox which accept srcdoc parameter.

**Comment by BrainwreckedTech** [comment]
I got this to work by changing the following line in `VideoService.php` for the `nico[]` array:

```
'embed' => '<iframe allowfullscreen="allowfullscreen" frameborder="0" width="%2$d" height="%3$d" src="https://embed.nicovideo.jp/watch/sm31502176?oldScript=1&amp;allowProgrammaticFullScreen=1" style="max-width: 100%;"></iframe>',
```

The only problem is, at the moment, at least for me, it seems playing embedded Nico Nico Douga videos does not work for Chrome/Chromium (yes, I tested both) under Linux.  Firefox under Linux works.  But that's not a result of my porting Nico Nico Douga's embedding code.  Their own code produces this behavior.

**Comment by BrainwreckedTech** [comment]
Created pull request #100

**Comment by BrainwreckedTech** [comment]
Created pull request #101 as #100 had a hard-coded smid.

**Comment by BrainwreckedTech** [comment]
TL;DR It was me, not Nico.  The code works fine.

Finally figured out the problem I was having with Nico embeds not working in Chrome.  I was blocking 3rd party cookies.  Last I regularly used Firefox, I had Firefox set to "Ask me first," But now Mozilla has done away with that setting and I fell back to Always.
